actiontext/lib/action_text/fixture_set.rb

```diff
@@ -1,7 +1,57 @@
 # frozen_string_literal: true
 
 module ActionText
+  # \Fixtures are a way of organizing data that you want to test against; in
+  # short, sample data.
+  #
+  # To learn more about fixtures, read the ActiveRecord::FixtureSet
```

Comment:
Can we add a link here?

Reply:
That sounds great! What's the rdoc syntax for that?

Reply:
http://docs.seattlerb.org/rdoc/RDoc/Markup.html#class-RDoc::Markup-label-Links